src/snowflake/snowpark/relational_grouped_dataframe.py

```diff
@@ -49,7 +43,7 @@ def _alias(expr: Expression) -> NamedExpression:
     else:
         return Alias(
             expr,
-            _strip_invalid_sf_identifier_chars(expr.sql.upper()),
+            expr.sql.upper().replace('"', ""),
```

Comment:
@sfc-gh-bli Is this a good fix from your understanding?

Reply:
I think, maybe, we should also remove `.replace('"', "")`

Reply:
for example, if a table has two columns, one is `"`, the other is `"""`, then we will meet this issue again.